test/e2e/predictor/test_triton.py

```diff
@@ -13,52 +13,42 @@
 # limitations under the License.
 
 import os
-from kubernetes import client
+
 import numpy as np
+import pytest
+from kubernetes import client
+from kubernetes.client import V1ResourceRequirements
+
 from kserve import KServeClient
-from kserve import constants
-from kserve import V1beta1PredictorSpec, V1beta1TransformerSpec
-from kserve import V1beta1TritonSpec
-from kserve import V1beta1InferenceServiceSpec
 from kserve import V1beta1InferenceService
+from kserve import V1beta1InferenceServiceSpec
 from kserve import V1beta1ModelSpec, V1beta1ModelFormat
-from kubernetes.client import V1ResourceRequirements, V1Container, V1ContainerPort
-import pytest
+from kserve import V1beta1PredictorSpec
+from kserve import V1beta1TritonSpec
+from kserve import constants
 from ..common.utils import KSERVE_TEST_NAMESPACE
 from ..common.utils import predict
 
 
-# TODO: don't skip after kserve/kserve#2972 is merged
-@pytest.mark.skip()
+@pytest.mark.fast
 def test_triton():
     service_name = 'isvc-triton'
     predictor = V1beta1PredictorSpec(
         min_replicas=1,
         triton=V1beta1TritonSpec(
             storage_uri='gs://kfserving-examples/models/torchscript',
-            ports=[V1ContainerPort(name="h2c", protocol="TCP", container_port=9000)],
             resources=V1ResourceRequirements(
                 requests={'cpu': '10m', 'memory': '128Mi'},
                 limits={'cpu': '100m', 'memory': '512Mi'},
             ),
         )
     )
-    transformer = V1beta1TransformerSpec(
-        min_replicas=1,
-        containers=[V1Container(
-                      image='kserve/image-transformer:'
-                            + os.environ.get("GITHUB_SHA"),
-                      name='kserve-container',
-                      resources=V1ResourceRequirements(
-                          requests={'cpu': '10m', 'memory': '128Mi'},
-                          limits={'cpu': '100m', 'memory': '512Mi'}),
-                      args=["--model_name", "cifar10", "--protocol", "grpc-v2"])]
-    )
```

Comment:
Any reason to remove the transformer?

Reply:
There was a bug in image transformer. I have fixed the bug and added transformer test.